capellambse/model/layers/la.py

```diff
@@ -59,7 +59,7 @@ class LogicalComponent(cs.Component):
 
     _xmltag = "ownedLogicalComponents"
 
-    functions = c.ReferencingProxyAccessor(
+    allocated_functions = c.ReferencingProxyAccessor(
```

Comment:
What about people that used `functions` before? That would break with this, but c0153237d0d4ce0474f4cb74f74df211dcb5da1f, which introduced this, is not marked as "breaking change".